sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/CertificateCredentialImpl.java

```diff
@@ -32,9 +32,8 @@
 
     CertificateCredentialImpl(MicrosoftGraphKeyCredentialInner keyCredential) {
         super(keyCredential);
-        if (keyCredential.customKeyIdentifier() != null && keyCredential.customKeyIdentifier().length > 0) {
-            this.name = new String(Base64.getMimeDecoder().decode(keyCredential.customKeyIdentifier()),
-                StandardCharsets.UTF_8);
+        if (keyCredential.displayName() != null) {
+            this.name = keyCredential.displayName();
```

Comment:
Due to the displayName logic chagned, it may be a runtime breaking change. User get credential created from previous package will get different name.